lib/Sema/CSFix.h

```diff
@@ -1958,6 +1961,24 @@ class SpecifyLabelToAssociateTrailingClosure final : public ConstraintFix {
   create(ConstraintSystem &cs, ConstraintLocator *locator);
 };
 
+/// Diagnose situations where we have key path missing a component.
+///
+/// \code
+/// let _ : KeyPath<A, B> = \A
+/// \endcode
+class AllowKeyPathMissingComponent final : public ConstraintFix {
```

Comment:
Same name here? I think that's better than "missing component" because the problem is that there are _no components_ :)

Reply:
Right, all fixed!

Reply:
Good to go?